aws/resource_aws_db_instance.go

```diff
@@ -220,6 +220,49 @@ func resourceAwsDbInstance() *schema.Resource {
 				},
 			},
 
+			"s3_import": {
```

Comment:
We should add `ForceNew: true` to all child attributes of `s3_import` as there is no "update" available for it. 😄

Reply:
Added this, though I could see long term an interesting debate.  if it's use as a spring-board to create a database, should be able to remove the backup file after the database is created.  SO in theory, deleting the block shouldn't cause a rebuild/recreate.  But I guess that can be done with ignore attributes block instead...